config-example

```diff
@@ -12,12 +12,11 @@ DAVUSER="user"
 # leave it empty, then you will be promted every time you use the script
 PASSWORD="123456"
```

Comment:
if we switch to the credantial file stored in .config
this is no longer needed right?

Reply:
Yes. I've fixed it.